src/routes/conversation/[id]/+server.ts

```diff
@@ -136,38 +138,53 @@ export async function POST({ request, locals, params, getClientAddress }) {
 		hashes = await Promise.all(files.map(async (file) => await uploadFile(file, conv)));
 	}
 
+	// can only call isContinue on the last message id
+	if (
+		isContinue &&
+		conv.messages.findIndex((message) => message.id === messageId) !== conv.messages.length - 1
```

Comment:
```suggestion
		conv.messages[conv.messages.length - 1].id !=== messageId
```

haven't tested it but wouldn't this simpler boolean statement work ?